aclk/aclk_query.c

```diff
@@ -427,7 +420,7 @@ static int aclk_execute_query_v2(struct aclk_query *this_query)
     mysep = strrchr(this_query->query, '/');
 
     // execute the query
-    t = aclk_web_api_request_v1(localhost, w, mysep ? mysep + 1 : "noop");
+    t = aclk_web_api_request_v1(localhost, w, mysep ? mysep + 1 : "noop", this_query->created);
```

Comment:
The return value `t` is not used. Actually there is another call to `aclk_web_api_request_v1`  L357 that drops the return value (the metrics are updated  within the aclk_web_api_request_v1)  - - consider switching to void?

Reply:
It is used here:
`buffer_sprintf(local_buffer, ",\"t-exec\": %llu,\"t-rx\": %llu,\"http-code\": %d", t, this_query->created, w->response.code);`
The aclk v2 adds fields so cloud has this information into the message.
ACLK v1 doesn't use this time